runner.rb

```diff
@@ -0,0 +1,10 @@
+require './lib/game.rb'
+
+def start
+  game = Game.new
+  game.welcome
+  game.setup
```

Comment:
Looking good we still need to get that loop back to the welcome message in there!